### PR TITLE
 stackrox: release v4.0 - bump image for go 19.9 to fix CVE

### PR DESCRIPTION
**Pull request by davdhacs**
Tom released 0.3.59 and 0.3.57.1: https://github.com/openshift/release/pull/39424
Images are available at https://quay.io/repository/stackrox-io/apollo-ci?tab=tags
stackrox-test-0.3.57.1 sha256:efed75ca306d17e14126c0e2700053bf2845d2dafb621d9779779d88ce1197b5
stackrox-test-0.3.59 sha256:7236f9aa229d3dc932b146e70f05d27941664b76bd41cd588678afcbf24b4fe1

### CODE REVIEW

**Comment by BradLugo** [comment]
/pj-rehearse pull-ci-stackrox-stackrox-release-4.0-gke-qa-e2e-tests pull-ci-stackrox-stackrox-release-4.0-ocp-4-12-operator-e2e-tests pull-ci-stackrox-stackrox-release-4.0-ocp-4-12-qa-e2e-tests pull-ci-stackrox-stackrox-release-4.0-ocp-4-12-ui-e2e-tests

**Comment by BradLugo** [comment]
@davdhacs can you add `stackrox: ` or the issue number (`ROX-XYZ: `) to the title of the PR please 🙂 

If these rehearsals pass (or at least get to a reasonable step in the tests and then flake out), it'll lgtm

File: ci-operator/config/stackrox/stackrox/stackrox-stackrox-release-4.0__ocp-4-13.yaml [hunk 1]
```diff
@@ -7,7 +7,7 @@ build_root:
   image_stream_tag:
     name: apollo-ci
     namespace: stackrox
-    tag: stackrox-test-0.3.58
+    tag: stackrox-test-0.3.59
```
Comment (gavin-stackrox) [inline ci-operator/config/stackrox/stackrox/stackrox-stackrox-release-4.0__ocp-4-13.yaml:10]: I think this should also be `0.3.57.1`. All the CI for a release branch should run in the same environment. This probably originates from a copy paste error. Similar to https://github.com/openshift/release/pull/39523.

**Review by gavin-stackrox — CHANGES_REQUESTED** [review]

**Review by gavin-stackrox — APPROVED** [review]

**Pull request merged**